src/apns_utils.erl

```diff
@@ -35,9 +35,10 @@
 -spec sign(binary()) -> binary().
 sign(Data) ->
   {ok, KeyPath} = application:get_env(apns, token_keyfile),
+  {ok, Openssl} = application:get_env(apns, openssl_path),
   Command = "printf '" ++
             binary_to_list(Data) ++
-            "' | openssl dgst -binary -sha256 -sign " ++ KeyPath ++ " | base64",
+            "' | " ++ Openssl ++ " dgst -binary -sha256 -sign " ++ KeyPath ++ " | base64",
```

Comment:
Line 41 is too long:             "' | " ++ Openssl ++ " dgst -binary -sha256 -sign " ++ KeyPath ++ " | base64",.